Sources/UtiliKit/Container/ContainerViewController.swift

```diff
@@ -9,6 +9,12 @@ import UIKit
 
 open class ContainerViewController: UIViewController {
     
+    // MARK: Subtypes
+    public enum PostTransitionBehavior {
+        case removeAllNonVisibleChildren
+        case removeAllNonVisibleChildrenExcept([AnyHashable])
```

Comment:
Could also have a `default` or `none` case here in then make the property non-optional?

Reply:
Done. I was on the fence about this when I originally worked on it. It actually ends up identical when we handle it in a switch, since optionals have a `.none` case anyway lol